include/seqan3/range/container/concatenated_sequences.hpp

```diff
@@ -342,15 +383,16 @@ class concatenated_sequences
     template <std::ranges::forward_range rng_type>
     concatenated_sequences & operator=(std::initializer_list<rng_type> ilist)
```

Comment:
maybe adapt the template name here as well, since you changed already above

Reply:
Sorry can't find a rename, in what should I rename this?

Reply:
Like in the initialiser list constructor? There you changed `rng_type` to `value_type_t`. And now in the assignment you don't, but I guess what you like to express is. Well, it is a generic type but please make note, that it should be `value_type`. Also I think as you said in the comments in the future this should only be `std::initializer_list<value_type>` and not some generic compatible type. That will always be troublesome through weird conversion sequences.

Reply:
Not completely sure what I should do, I assume you want `rng_type` renamed to `value_type_t`.